do_init.go

```diff
@@ -20,6 +21,19 @@ func doInitialize(fs *flag.FlagSet, argv []string) error {
 		// Setting apikey via environment variable should be supported or not?
 		return fmt.Errorf("-apikey option is required")
 	}
+
+	// make the config file directory if not exist
+	root := filepath.Dir(*conffile)
+	if _, err := os.Stat(root); err != nil {
+		if !os.IsNotExist(err) {
+			return err
+		}
+		err := os.MkdirAll(root, 0744)
```

Comment:
Is there any reason to choose 0744, not 0755?
Currently we use 0755 for newly created directories (explicitly in some cases, or implicitly in many cases). So I  prefer 0755 here if there's no special reason.

Reply:
OK. thank you for reviewing